public/app/features/explore/TraceView/components/ScrollManager.tsx

```diff
@@ -268,6 +269,7 @@ export default class ScrollManager {
 
   destroy() {
     this._trace = undefined;
+    /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Are you planning to fix typing in upcoming PR/s? Do you have a task in the backlog to address it?

Reply:
FWIW we should probably just remove `eslint-disable @typescript-eslint/no-explicit-any` and add an entry to betterer results instead, so they'll be tracked with all others `any` usage